MapboxMobileEvents/MMEAPIClient.m

```diff
@@ -207,6 +207,29 @@ - (void)processRequest:(NSURLRequest *)request {
             }
 
             NSUserDefaults.mme_configuration.mme_configUpdateDate = MMEDate.date;
+
+            NSString *digest = nil;
+            NSString *digestHeader = httpResponse.allHeaderFields[@"Digest"];
```

Comment:
nit: Should we use here `valueForHTTPHeaderField` method?

Reply:
'valueForHTTPHeaderField:' is only available on iOS 13.0 or newer, so I am not sure whether we should use that or not.